mypy_self_check.ini

```diff
@@ -7,7 +7,12 @@ no_implicit_optional = True
 disallow_any = generics, unimported
 warn_redundant_casts = True
 warn_unused_ignores = True
+warn_unused_configs = True
 
 # historical exception
 [mypy-mypy.semanal]
 strict_optional = False
+
+# needs py2 compatibility
```

Comment:
I'm not sure this is true, but the file is syntactically valid in Python 2 and it tests a piece of code (mypy_extensions.py) that is supposed to work in Python 2. @ilevkivskyi can you confirm.

Reply:
Yes, this file supposed to run on both Python 3 _and_ Python 2. There are still some problem with type checking it, but there is a separate issue for this https://github.com/python/mypy/issues/2755